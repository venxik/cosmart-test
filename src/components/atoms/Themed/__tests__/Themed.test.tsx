import React from 'react';
import renderer from 'react-test-renderer';
import { Text, View } from '../Themed';

describe(`Themed test`, () => {
  it(`renders Text correctly`, () => {
    const tree = renderer.create(<Text>Snapshot test!</Text>).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders View correctly`, () => {
    const tree = renderer.create(<View>Snapshot test!</View>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
