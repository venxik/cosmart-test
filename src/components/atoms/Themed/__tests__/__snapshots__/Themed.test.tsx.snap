// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Themed test Text match snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`Themed test View match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "borderColor": "#2f95dc",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</View>
`;
